google-feeds.html

```diff
@@ -215,6 +230,19 @@
           this._feedsCallbacks.push(callback);
         }
       }
+    },
+
+    _format: function() {
```

Comment:
I think cannot be a computed property because the format values from Feed API hasn't been loaded so when the function is executed *google* var is not defined yet.